kitsune/community/api.py

```diff
@@ -0,0 +1,330 @@
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+from elasticutils import F
+from rest_framework import views, fields
+from rest_framework.response import Response
+
+from kitsune.questions.models import AnswerMetricsMappingType
+from kitsune.sumo.api import GenericAPIException
+from kitsune.users.models import UserMappingType
+from kitsune.wiki.models import RevisionMetricsMappingType
+
+
+# This should be the higher than the max number of contributors for a
+# section.  There isn't a way to tell ES to just return everything.
```

Comment:
Ooh, nice. I stole that comment, and the general approach of getting metric objects, and then getting user objects, from some stuff r1cky wrote for the first iteration of this feature.